### DISCUSSION TITLE
Result still SVG when trying to create PNG based on examples

### DISCUSSION THREAD

**Discussion by vaporizel** [chillerlan/php-qrcode#242] [Q&A]
i'm trying to run this example :
https://github.com/chillerlan/php-qrcode/blob/main/examples/image.php

its doesnt show up, im using header image/png, because the actual image is printed using SVG tag

**Comment by codemasher** [comment]
Hey, see https://github.com/chillerlan/php-qrcode/discussions/241#discussioncomment-8461662
